ssd1306/ssd1306.c

```diff
@@ -36,6 +40,11 @@ void ssd1306_Reset(void) {
 void ssd1306_WriteCommand(uint8_t byte) {
     HAL_GPIO_WritePin(SSD1306_CS_Port, SSD1306_CS_Pin, GPIO_PIN_RESET); // select OLED
     HAL_GPIO_WritePin(SSD1306_DC_Port, SSD1306_DC_Pin, GPIO_PIN_RESET); // command
+    while(SSD1306_SPI_PORT.State != HAL_SPI_STATE_READY); 	// Wait until previous transmit is done.
+                                                            // Before transmit a new sequence,
+                                                            // neither HAL_SPI_Transmit_DMA() nor HAL_SPI_Transmit() 
+                                                            // confirms the previous transmit is done in
+                                                            // STM32F1xx HAL driver V1.1.7, Oct. 20, 2020
     HAL_SPI_Transmit(&SSD1306_SPI_PORT, (uint8_t *) &byte, 1, HAL_MAX_DELAY);
```

Comment:
Shouldn't it be another ifdef here?

Reply:
Same as [above](https://github.com/afiskon/stm32-ssd1306/pull/42/files/861f7e42293779e1272632e7110c7f1032a4085a#r654851563).